models.js

```diff
@@ -0,0 +1,12 @@
+import mongoose from 'mongoose'
+
+export const Shows = mongoose.model('Shows', {
+  "show_id": Number,
+  "title": String,
+  "director": [String],
+  "cast": [String],
+  "country": [String],
+  "release_year": Number,
+  "description": String,
+  "type": String,
```

Comment:
It will be safer for you if you add `required: true` to some fields. this will prevent from creating empty entries in the database ;)